cumulus_library_catalog/reference_sql/icd10_annotated_counts.sql

```diff
@@ -44,24 +44,26 @@ CREATE TABLE catalog__count_icd10_diagnoses AS (
 
     SELECT
         p.cnt_subject_ref AS cnt,
-        p.code,
-        j.chapter_code,
-        j.chapter_str,
-        j.block_code,
-        j.block_str,
-        j.category_code,
-        j.category_str,
-        j.subcategory_1_code,
-        j.subcategory_1_str,
-        j.subcategory_2_code,
-        j.subcategory_2_str,
-        j.subcategory_3_code,
-        j.subcategory_3_str,
-        j.extension_code,
-        j.extension_str
+        p."code",
+        j."chapter_code",
+        j."chapter_str",
+        j."block_code",
+        j."block_str",
+        j."category_code",
+        j."category_str",
+        j."subcategory_1_code",
+        j."subcategory_1_str",
+        j."subcategory_2_code",
+        j."subcategory_2_str",
+        j."subcategory_3_code",
+        j."subcategory_3_str",
+        j."extension_code",
+        j."extension_str"
     FROM powerset AS p
         JOIN "umls"."icd10_hierarchy" j ON p.code = j.leaf_code
     WHERE
-        system = 'http://hl7.org/fhir/sid/icd-10-cm'
```

Comment:
noting that this was not up to date with the actual checked in builder on the previous commit